### PR TITLE
[PL-46862] PL 1.24.7 Release Notes

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-46862](https://harness.atlassian.net/browse/PL-46862)

https://harness.atlassian.net/wiki/spaces/PD/pages/21637005335/NG+Release+SAAS+February+5th+2024+for+1.24.x-+Build

https://65c4bc926564570a74526be5--harness-developer.netlify.app/release-notes/platform

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot. 

[PL-46862]: https://harness.atlassian.net/browse/PL-46862?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: release-notes/platform.md [hunk 1]
```diff
@@ -82,6 +82,40 @@ The following deprecated API endpoints are longer supported:
...
+
+   ```
+
+- Support added for encrypted assertions in the SAML response. (PL-43353)
+
+#### Fixed issues
+
+- Users were unable to change the authentication mechanism of a secret manager from a non-password based to a password based mechanism due to a bug in the secret manager update API. (PL-46657)
+
+   This issue has been fixed, enabling the modification of the authentication mechanism for secret managers to a password based mechanism in this release.
```
Comment (brian-f-harness) [inline release-notes/platform.md:111]:
```suggestion
   This issue has been fixed, enabling the modification of the authentication mechanism for secret managers to a password-based mechanism.
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -82,6 +82,40 @@ The following deprecated API endpoints are longer supported:
...
+   maxBackupFileCount: 10 #max count of files
+   totalFileSizeCap: 1GB
+
+   ```
+
+- Support added for encrypted assertions in the SAML response. (PL-43353)
+
+#### Fixed issues
+
+- Users were unable to change the authentication mechanism of a secret manager from a non-password based to a password based mechanism due to a bug in the secret manager update API. (PL-46657)
```
Comment (brian-f-harness) [inline release-notes/platform.md:109]:
```suggestion
- Users weren't able to change the authentication mechanism of a secret manager from a non-password-based to a password-based mechanism due to a bug in the secret manager update API. (PL-46657)
```

File: release-notes/platform.md [hunk 3]
```diff
@@ -82,6 +82,40 @@ The following deprecated API endpoints are longer supported:
...
+   fileLogging:
+   enabled: true
+   logFilename: /opt/harness/logs/pod.log #provide log filename
+   maxFileSize: 50MB #max single file size, for log archiving
+   maxBackupFileCount: 10 #max count of files
+   totalFileSizeCap: 1GB
+
+   ```
+
+- Support added for encrypted assertions in the SAML response. (PL-43353)
```
Comment (brian-f-harness) [inline release-notes/platform.md:105]:
```suggestion
- Added support for encrypted assertions in the SAML response. (PL-43353)
```

File: release-notes/platform.md [hunk 4]
```diff
@@ -82,6 +82,40 @@ The following deprecated API endpoints are longer supported:
 
 ## February 2024
 
+### Version 1.24.4 <!--  February 12, 2024 -->
+
+#### New features and enhancements
+
+- Users can now enable file logging for supported services through override in On-prem installations. (PL-44211)
```
Comment (brian-f-harness) [inline release-notes/platform.md:89]:
```suggestion
- You can now enable file logging for supported services through override in Harness Self-Managed Enterprise Edition (On-prem) installations. (PL-44211)
```

File: release-notes/platform.md [hunk 5]
```diff
@@ -82,6 +82,40 @@ The following deprecated API endpoints are longer supported:
...
+
+   To enable file logging add the following to your override in the global section:
```
Comment (brian-f-harness) [inline release-notes/platform.md:91]:
```suggestion
   To enable file logging, add the following to your override in the global section:
```

File: release-notes/platform.md [hunk 6]
```diff
@@ -82,6 +82,40 @@ The following deprecated API endpoints are longer supported:
...
+
+   ```
```
Comment (brian-f-harness) [inline release-notes/platform.md:93]:
```suggestion
   ```yaml
```

File: release-notes/platform.md [hunk 7]
```diff
@@ -82,6 +82,40 @@ The following deprecated API endpoints are longer supported:
...
+
+   global:
```
Comment (brian-f-harness) [inline release-notes/platform.md:95]: Recommend fixing the YAML spacing. https://harness.atlassian.net/browse/PL-44211?focusedCommentId=666095

**Review by brian-f-harness — APPROVED** [review]

**Review by pratmit — APPROVED** [review]
LGTM

**Pull request merged**